packages/notifi-web-push-service-worker/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@notifi-network/notifi-web-push-service-worker",
+  "version": "0.0.1",
```

Comment:
@eric-notifi do we have to align versions to other packages in notifi-sdk-ts here?

Reply:
Well, yeah I think, we can put the updated version `1.1.2` here for now. Once it gets into the `lerna publish` process, lerna will handle the version updating for us.

Reply:
FYI, we need to align this version with the current updated version so that packages in mono-repo and refer to each other